content/en/getting-started/glossary.md

```diff
@@ -9,4 +9,30 @@ cascade:
 hide_readingtime: true
 ---
 
-Glossary
\ No newline at end of file
+This page defines LocalStack terminologies that you will encounter throughout the documentation.
+
+## LocalStack terms
+
+| Term | Definition |
```

Comment:
NIT: the `Term` for each definition is in capitalized, except for `LocalStack container` and `External service ports`